### DISCUSSION TITLE
Boot problem

### DISCUSSION THREAD

**Discussion by witzker** [ej52/proxmox-scripts#130] [General]
When I boot Nginx Proxy Manager, I get this:
![grafik](https://github.com/ej52/proxmox-scripts/assets/4120266/cd37deef-41c1-44e8-b945-3e4fa341ecc6)

 I did an: apk update , apk upgrade

Then I run the installation script for Alpine again 

> Alternative Usage
> 
> If you are not using proxmox or want to install this on a existing Alpine box, you can run the setup script itself.
> 
> Note: Only Alpine, Debian and Ubuntu are currently supported by this script
> 
> wget --no-cache -qO - https://raw.githubusercontent.com/ej52/proxmox/main/lxc/nginx-proxy-manager/setup.sh | sh

I get this at the end:
```
[error] Starting services
/tmp/npm_install.sh: line 250: rc-service openresty start
4
configuration file /etc/nginx/nginx.conf test failed
openresty failed to start
```
As I'm not clever with all this server stuff and just learning pls. Help to solve this
many THX.

**Comment by ej52** [comment]
@witzker Please try the latest script.
